nixos/modules/services/web-apps/invidious.nix

```diff
@@ -114,7 +114,7 @@ let
       check_tables = true;
 
       db = {
-        user = lib.mkDefault "kemal";
+        user = lib.mkDefault "invidious";
```

Comment:
If you do it this way, BTW, you have to notice that you will break every existing setups that defaulted on kemal.

Reply:
I thought about using `lib.versionAtLeast config.system.stateVersion "23.11"` for this to only default to `"invidious"` for newer setups but 1) the postgres module will just create the new user and change the owner of the existing db to the new user without breakage and 2) externally provisioned databases most likely configure this explicitly anyway. I've added a corresponding entry to the release notes for that. If you want, we can still switch this to check the state version?

Reply:
I think it would be good to have the `stateVersion` style stuff if possible.

Reply:
Done!